### PR TITLE
RBAC: Adding OpenLDAP to admin node

### PR DESCRIPTION
**Pull request by robdaemon**

### CODE REVIEW

**Comment by robdaemon** [comment]
Related: kubic-project/salt#192

**Comment by robdaemon** [comment]
Needs another change to populate the main LDAP group via LDIF file, coming soon. Moving to sles-based container will come as a separate PR.

**Comment by robdaemon** [comment]
Related: kubic-project/velum#286

**Comment by robdaemon** [comment]
@kiall @grahamhayes this is ready to go.

File: public.yaml [hunk 1]
```diff
@@ -245,6 +245,26 @@ spec:
...
+    - name: LDAP_PORT
+      value: "389"
+    - name: LDAP_GROUP_BASE_DN
+      value: ou=Groups,dc=infra,dc=caasp,dc=local
+    - name: LDAP_PEOPLE_BASE_DN
+      value: ou=People,dc=infra,dc=caasp,dc=local
+    - name: LDAP_BIND_DN
+      value: cn=admin,dc=infra,dc=caasp,dc=local
+    - name: LDAP_BIND_PASSWORD
+      value: admin
```
Comment (ereslibre) [inline public.yaml]: Is this something we can generate?

Comment (robdaemon) [reply public.yaml]: Yep! That was the reason behind my email to caasp-devel :)

**Pull request merged**